Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.2'
```

Comment:
code rails dùng dấu `"` thay cho `'` e nhé, thay thế toàn bộ đi e, tham khảo các bước cần check sau mỗi chapter tại đây nhé https://docs.google.com/spreadsheets/d/1ME3lk9-kYoQwC-3Exrq9ZkA8F6o0ZJxgGH8PKru7Bug/edit#gid=1196249713